packages/amplify-graphql-model-transformer/src/graphql-model-transformer.ts

```diff
@@ -127,10 +110,10 @@ export const directiveDefinition = /* GraphQl */ `
   }
 `;
 
-type ModelTransformerOptions = {
-  EnableDeletionProtection?: boolean;
-  SyncConfig?: SyncConfig;
-};
+const MYSQL_DB_TYPE = 'MySQL';
```

Comment:
Probably better to move these constants to transformer core to be used across transformers.